sdk/metricsadvisor/ai-metrics-advisor/test/public/util/recordedClients.ts

```diff
@@ -33,7 +33,7 @@ const blobTemplate = "blob_template";
 const replaceableVariables: { [k: string]: string } = {
   AZURE_CLIENT_ID: "azure_client_id",
   AZURE_CLIENT_SECRET: "azure_client_secret",
-  AZURE_TENANT_ID: "azure_tenant_id",
+  AZURE_TENANT_ID: "72698866-8641-4147-9144-24744011447",
```

Comment:
oh ok ok, so this is the right tenant. I see.

Reply:
The PR description mentions a different tenant 😮

Reply:
I can update the PR description - but just curious if we want to keep the same tenant id for recordings across all packages?

Reply:
I was mainly confused between the PR description and the PR values.

But if you ask me, I think it would be good to use the same fake tenant as we’ve used before.